src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,107 @@
+            ResultSet resultSet = getAllStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(getManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all manufacturers from DataBase " + e);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String createQuery = "INSERT INTO car_db (name, country) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement = connection.prepareStatement(createQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, manufacturer.getName());
+            createStatement.setString(2, manufacturer.getCountry());
+            createStatement.executeUpdate();
+            ResultSet resultSet = createStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t insert manufactures to DataBase " + e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateQuery = "UPDATE car_db SET name = ?, country = ? "
+                + "WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateStatement = connection.prepareStatement(updateQuery)) {
+            updateStatement.setString(1, manufacturer.getName());
+            updateStatement.setString(2, manufacturer.getCountry());
+            updateStatement.setLong(3, manufacturer.getId());
+            if (updateStatement.executeUpdate() > 0) {
+                return manufacturer;
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all manufacturers from DataBase " + e);
+        }
+        throw new DataProcessingException(String.format("No manufacturer %s in db", manufacturer));
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE car_db SET is_deleted = true WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteStatement = connection.prepareStatement(deleteQuery)) {
+            deleteStatement.setLong(1, id);
+            return deleteStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t delete manufactures from DataBase " + e);
+        }
+    }
+
+    private static Manufacturer getManufacturer(ResultSet resultSet) throws SQLException {
```

Comment:
Method should not be static
```suggestion
    private Manufacturer getManufacturer(ResultSet resultSet) throws SQLException {
```